app/views/discord_engine/interactions/say_hi.jbuilder

```diff
@@ -1 +1,6 @@
-json.partial! 'discord_engine/partials/single_line', resolver: @resolver
+json.partial!(
+  'discord_engine/partials/single_line',
+  type: @resolver.callback.type,
+  flags: @resolver.respond_to?(:flags) ? @resolver.flags : nil,
```

Comment:
https://apidock.com/rails/Object/try
```suggestion
  flags: @resolver.try(:flags),
```

Reply:
Same here, maybe we don't need to `try` the method.
```suggestion
  flags: nil,
```

Reply:
Yeah I agree with this last one